### DISCUSSION TITLE
Make the dedicated text object have outlines, and the ability to layer shadows and outlines

### DISCUSSION THREAD

**Discussion by thepwds** [olive-editor/olive#1933] [Ideas]
It would be really nice if you could put the shadow in front of the border to create a cushy effect. I have done it before and it looks great!
Plus, merge the Rich Text and the Text Effect to be generally one object, but i really would like the feature to layer different border and shadow effects.

**Comment by GUY-BBS** [comment]
Hi, you can create all kinds of text effects using nodes. I produced the video below a couple of months ago to demonstrate drop shadow and outline effects. Since then, the new text features have been introduced in Olive making the range and style of captions even more comprehensive. I create my graphic node set-ups on transparent png images of 1080 by 1920 pixels, which can be saved as text files and used in other projects or transferred to other computers. Ths means I only need to create my node setup once.
https://user-images.githubusercontent.com/49677111/168478787-a5ceee20-273f-4ca9-9c91-61df515b52f2.mp4

**Comment by EviTRea** [comment]
What about having a default stroke effect node attached, or an easy way to figure out texts should be coupled with other nodes?
